.travis.yml

```diff
@@ -7,12 +7,13 @@ php:
     - 5.6
     - 7.0
     - 7.1
+    - 7.2
+    - 7.3
 
 before_install:
     - if [ "$TRAVIS_PHP_VERSION" != "hhvm" ]; then composer require satooshi/php-coveralls:^1.0 --no-update; fi;
 
 install:
-    - composer require phpunit/phpunit --no-update
```

Comment:
This is a step in right direction, but you're not quite there yet.

At time, when I was creating this `.travis.yml` file I wasn't aware, that Travis CI already has built-in PHPUnit version and it should be used instead. Since PHPUnit itself tends to break BC a lot with each feature release using version from Travis CI all the time won't help us much (especially on PHP 7.2+, where PHPUnit 8 is used). You can try yourself to verify it.

You can use approach from https://github.com/console-helpers/svn-buddy/blob/master/.travis.yml as a start point and modify it as needed. The goal is to to use best compatible PHPUnit version based on PHP version.

Reply:
I will scratch that `.travis.yml` file that you mention.